tests/cli/test_package.py

```diff
@@ -17,7 +17,7 @@
 from uhu.core.utils import dump_package, load_package
 from uhu.updatehub.api import UpdateHubError
 from uhu.utils import LOCAL_CONFIG_VAR, SERVER_URL_VAR
-
+from tempfile import mktemp
```

Comment:
You should use this.

Reply:
I tried that and resorted to regular files lastly. But yeah, let me try once again.